dcos/dcos-ui
# DCOS_OSS-867: Unspecific GUI Error Message

Display an error even when the response is empty.

Closes https://jira.mesosphere.com/browse/DCOS_OSS-867

## Testing
1. In Chrome, enable request blocking and block `localhost:4200/package/search`.
2. Go to Catalog tab.
3. Verify that the error message is not empty.
4. Verify that the added test makes sense.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
### Before
![Снимка от 2019-06-27 12-52-18](https://user-images.githubusercontent.com/40791275/60262942-185b9300-98e8-11e9-8542-a901045227ba.png)

### After
![Снимка от 2019-06-27 16-44-29](https://user-images.githubusercontent.com/40791275/60271354-db4ccc00-98fa-11e9-9f2f-6b98b2600ebf.png)
